### PR TITLE
fix: Change service key from device-bacnet to device-bacnet-ip

### PR DESCRIPTION
**Pull request by FelixTing**
**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

fix: #397 

## Testing Instructions

`make run ds-bacnet-ip`
Verify if the BACnet device service runs normally.

### CODE REVIEW

**Review by lenny-intel — APPROVED** [review]
LGTM, please also make same change on the `minnesota` branch

**Comment by cloudxxx8** [comment]
> LGTM, please also make same change on the `minnesota` branch

@lenny-intel the current v3.0 bacnet doesn't work anyway.  may we fix it when we have 3.0.1 image?

File: compose-builder/Makefile [hunk 1]
```diff
@@ -41,7 +41,7 @@ define OPTIONS
  - arm64 no-secty dev app-dev device-dev ui-dev delayed-start -
  - nats-bus mqtt-bus mqtt-broker mqtt-verbose -
  - taf-secty taf-no-secty taf-perf taf-perf-no-secty -
- - ds-onvif-camera  ds-usb-camera ds-bacnet ds-modbus ds-mqtt ds-rest ds-snmp ds-virtual ds-llrp ds-coap ds-gpio -
+ - ds-onvif-camera  ds-usb-camera ds-bacnet-ip ds-modbus ds-mqtt ds-rest ds-snmp ds-virtual ds-llrp ds-coap ds-gpio -
```
Comment (lenny-intel) [inline compose-builder/Makefile:44]: Must also update option name in the README.

Should we also have an option for `ds-bacnet-ip`

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Comment by lenny-intel** [comment]
@FelixTing , please make sure the generated compose has the correct ``device-bacnet-ip` name in the volume mount for the vault token

File: compose-builder/Makefile (hunk 1 above)
Comment (FelixTing) [reply compose-builder/Makefile:44]: Will do. And I would also like to add an option for `ds-bacnet-mstp`.

**Comment by FelixTing** [comment]
> @FelixTing , please make sure the generated compose has the correct `device-bacnet-ip` name in the volume mount for the vault token

Verified. There is no problem with the volume mount.

**Comment by FelixTing** [comment]
recheck

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — DISMISSED** [review]
LGTM once conflict are resolved

**Comment by FelixTing** [comment]
rebased

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**